extensions/amp-user-notification/0.1/amp-user-notification.css

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+amp-user-notification {
+  visibility: hidden;
+  background: inherit;
+  padding: 2px;
+  border: 1px solid rgba(0, 0, 0, .1);
+  border-radius: 6px;
+  box-shadow: 0 8px 16px rgba(0, 0, 0, .25);
+  overflow: hidden;
+  min-height: 30px;
+  background: rgba(255, 255, 255, .60);
+  margin-bottom: 8px;
+  padding: 4px 4px;
+}
+
+amp-user-notification.amp-active {
+  visibility: visible;
+}
+
+amp-user-notification.amp-hidden {
+  display: none;
+}
+
+
+.-amp-glasspane {
```

Comment:
@cramforce do we want to bring this back so that multiple notifications are inside this fixed container?